lib/test-endpoint.sh

```diff
@@ -8,7 +8,7 @@ DD_API_SITE=$DD_US_API_SITE
 DD_USE_EU=false
 
 DD_DEFAULT_HTTPS_EU_ENDPOINT="agent-http-intake.logs.datadoghq.eu:443"
-DD_DEFAULT_HTTPS_US_ENDPOINT="agent-http-intake.logs.datadoghq.com:10516"
+DD_DEFAULT_HTTPS_US_ENDPOINT="agent-http-intake.logs.datadoghq.com:443"
 DD_DEFAULT_TCP_EU_ENDPOINT"agent-intake.logs.datadoghq.eu:443"
 DD_DEFAULT_TCP_US_ENDPOINT="agent-intake.logs.datadoghq.com:10516"
```

Comment:
Should we link this to some log agent doc?

Reply:
Good idea. Added a line above this section with a link to the logs docs.